qualification/conftest.py

```diff
@@ -531,7 +531,7 @@ async def receive_baseline_correlation_products(
     receiver = BaselineCorrelationProductsReceiver(
         cbf=cbf,
         stream_name="baseline-correlation-products",
-        core=core_allocator.allocate(1)[0],
+        cores=core_allocator.allocate(4),
```

Comment:
I gather this is more because how the cores are 'grouped' (to be spread across CCDs)
* i.e. Grab a whole group of four anyway,
* Rather than "we know we will always need/use four" (?)

Reply:
I didn't particularly think about the CCDs. I wanted a power of 2 (to evenly split the multicast groups), and I was worried that the gains from going from 1 to 2 might be wiped out by the overheads of the cores coordinating with each other, so I went for 4.

Additionally, katsdpcontroller always makes at least 4 XB-engines, so we're guaranteed to have something for each core to do.